deepmd/utils/argcheck.py

```diff
@@ -2305,6 +2305,10 @@ def model_args(exclude_hybrid=False):
 def standard_model_args() -> Argument:
     doc_descrpt = "The descriptor of atomic environment."
     doc_fitting = "The fitting of physical properties."
+    doc_model_branch_alias = (
+        "The alias of this model branch. This is only used in the multi-task model."
+    )
+    doc_info = "The information of this model branch. This is only used in the multi-task model."
```

Comment:
After reading the documentation, I cannot get how to set these two arguments. (and I think no one knows how to set them by only reading the documentation)